src/test/java/roomescape/MissionStepTest.java

```diff
@@ -15,20 +16,26 @@
 public class MissionStepTest {
 
     @Test
-    void 일단계() {
+    @DisplayName("/로 요청 시 index.html과 200 statusCode 반환하는지 테스트합니다.")
```

Comment:
```suggestion
    @DisplayName("/로 요청 시 index.html과 200 statusCode 반환한다.")
```

`테스트합니다.`불필요한 설명인 것 같습니다.

Reply:
넵 수정하겠습니다